general/server/live-tv-setup.md

```diff
@@ -0,0 +1,85 @@
+---
+uid: server-live-tv-setup
+title: Live TV Setup Guide
+---
+
+## Add a TV tuner to Jellyfin (Automatic Discovery)
+
+Click on the Admin Panel Icon in the top right corner (1)
+Click 'Live TV' (2) under the 'Live TV' section
+Click the '+' button (3) next to 'Tuner Devices'
+
+
+![](/images/live-tv-setup-tuner1.png)

```

Comment:
I believe all these image links should be without the leading `/`, it should be `~/` for the root of the repo. That also allows DocFX to properly rewrite those links, and check them for us.

https://dotnet.github.io/docfx/tutorial/links_and_cross_references.html#relative-path-vs-absolute-path